commands/db.bee.inc

```diff
@@ -348,3 +378,17 @@ function dbq_bee_callback($arguments, $options) {
     bee_message($e->getMessage(), 'error');
   }
 }
+
+/**
+ * Checks whether a command exists in the PATH, and if so, returns the full
+ * path. Returns null if no such command is available.
+ */
+function executable_for($cmd) {
+  foreach (explode(":", getenv('PATH')) as $dir) {
+    $candidate = "$dir/$cmd";
+    if (is_executable($candidate)) {
+      return $candidate;
+    }
+  }
+  return null;
```

Comment:
Please note
https://docs.backdropcms.org/php-standards#naming-conventions

> Constants should always be all-uppercase, with underscores to separate words. This includes pre-defined PHP constants like TRUE, FALSE, and NULL

```suggestion
  return NULL;
```

However, we don't normally use `NULL` as a return value. `FALSE` is simpler as the check for it in this case would be something like:
```php
if (!$cmd_bin) {
```

```suggestion
  return FALSE;
```